File: Graduway.API/Graduway.Web/Graduway.DAL/Entities/Employee.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Graduway.DAL.Entities
{
    public class Employee
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int? Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Department { get;set; }
        public string Title { get; set; }

        public List<Task> Tasks { get; set; }
    }
}

File: Graduway.Client/graduway-app/src/app/Models/employee.ts
import Task from './task';

export interface IEmployee {
  id: number;
  firstName: string;
  lastName: string;
  address: string;
  department: string;
  tasks: Task[];
  title: string;
}

class Employee {
  id: number;
  firstName: string;
  lastName: string;
  address: string;
  department: string;
  tasks: Task[];
  title: string;

  constructor(data?: Partial<IEmployee>) {
    this.id = data?.id;
    this.firstName = data?.firstName;
    this.lastName = data?.lastName;
    this.address = data?.address;
    this.department = data?.department;
    this.tasks = data?.tasks;
    this.title = data?.title;
  }
}

export default Employee;
